test/src/test/java/org/springframework/security/test/web/servlet/request/SecurityMockMvcRequestPostProcessorsCsrfTests.java

```diff
@@ -143,6 +147,25 @@ public void csrfWithWrappedRequest() throws Exception {
 		// @formatter:on
 	}
 
+	// gh-4016
+	@Test
+	public void csrfWhenUsedThenNotDoesNotImpactOriginalRepository() throws Exception {
```

Comment:
Typo: csrfWhenUsedThen**Not**DoesNotImpactOriginalRepository
Chaneg to: csrfWhenUsedThenDoesNotImpactOriginalRepository